### PR TITLE
Fix partial error messages being thrown during unit tests

### PR DESCRIPTION
**Pull request by adammahmood**
### What is the context of this PR?
- Fixes https://github.com/ONSdigital/eq-author-app/issues/124

### How to review 
- Ensure tests pass
- Ensure issue https://github.com/ONSdigital/eq-author-app/issues/124 is fixed

### CODE REVIEW

File: eq-author/src/tests/setup/throwOnConsole.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-console, import/unambiguous */
 const actualConsoleError = console.error.bind(console);
 
-console.error = jest.fn(function(msg) {
-  actualConsoleError(msg);
+console.error = jest.fn(function(msg, ...rest) {
+  actualConsoleError(msg, rest);
```
Comment (tgandrews) [inline eq-author/src/tests/setup/throwOnConsole.js]: Should this spread as well?
```suggestion
  actualConsoleError(msg, ...rest);
```

**Comment by rmccar** [comment]
Also tried adding Tom's suggested change but still got the same error

File: eq-author/src/tests/setup/throwOnConsole.js (hunk 1 above)
Comment (adammahmood) [reply eq-author/src/tests/setup/throwOnConsole.js]: Yes.  I think it's better to spread here as well.

**Comment by adammahmood** [comment]
This PR fixes the `console.error` so it can utilise any available substitution parameters and log error messages with these parameters substituted as per the [Console API](https://developer.mozilla.org/en-US/docs/Web/API/console#Outputting_text_to_the_console)

Unfortunately, it does not resolve the message `Error` throws as this would involve complex string substitution.

**Pull request merged**